sphinx_gallery/gen_gallery.py

```diff
@@ -41,6 +41,7 @@
     'filename_pattern': re.escape(os.sep) + 'plot',
     'ignore_pattern': r'__init__\.py',
     'examples_dirs': os.path.join('..', 'examples'),
+    'reset_argv': lambda gallery_conf, script_vars: [],
```

Comment:
Unstable repr:
```
(base) bunker:~ larsoner$ python -c "print(lambda x: None)"
<function <lambda> at 0x7ffdf80cd160>
(base) bunker:~ larsoner$ python -c "print(lambda x: None)"
<function <lambda> at 0x7f95b80cd160>
```
This needs to be a class with a stable `repr` to avoid always rebuilding docs for people who otherwise don't specify a value.

Reply:
Thank you, I missed that. Fixed it not now 8f1f10c0e6d16b25d5f73b9dbc4948485ae242f7